evm/deploy-contract.js

```diff
@@ -174,6 +172,7 @@ program.addOption(new Option('-n, --chainNames <chainNames>', 'chain names').mak
 program.addOption(new Option('-p, --privateKey <privateKey>', 'private key').makeOptionMandatory(true).env('PRIVATE_KEY'));
 program.addOption(new Option('-s, --salt <salt>', 'salt to use for create2 deployment'));
 program.addOption(new Option('-v, --verify', 'verify the deployed contract on the explorer').env('VERIFY'));
+program.addOption(new Option('-ve, --verifyEnv <verifyEnv>', 'environment for contract verification'));
```

Comment:
why is this needed? we already have `--env`

Reply:
good point